src/ripple/rpc/handlers/AccountLines.cpp

```diff
@@ -86,12 +85,10 @@ Json::Value doAccountLines (RPC::Context& context)
         return result;
 
     std::string strIdent (params[jss::account].asString ());
-    bool bIndex (params.isMember (jss::account_index));
-    int iIndex (bIndex ? params[jss::account_index].asUInt () : 0);
     RippleAddress rippleAddress;
 
-    Json::Value const jv (RPC::accountFromString (ledger, rippleAddress, bIndex,
-        strIdent, iIndex, false, context.netOps));
+    Json::Value const jv (RPC::accountFromString (rippleAddress,
```

Comment:
The ones named `jv` I tagged `auto`.  The ones named `jvAccepted` I renamed to `error`.  In both cases, that made the unbroken lines short enough.  Calls with each argument on its own line I left alone.